acm/src/main/java/org/literacybridge/acm/store/Committable.java

```diff
@@ -83,7 +82,11 @@ public final void commit(Transaction t) {
         }
     }
 
-    public abstract void doCommit(Transaction t) throws IOException;
+    /**
```

Comment:
I'm mostly objecting to "doCommit()" returning "true" if "the commit resulted in adding an object". If the caller of doCommit() needs to know what the commit did, the possibilities are richer than must "was an add/wasn't an add". 

Also, does this mean that a transaction can only add one object, or change one object, or delete one object? But nothing any more complex? (Just curious.)